src/App.tsx

```diff
@@ -59,6 +61,8 @@ export default function App() {
           tax: amount,
         };
       }
+      default:
```

Comment:
forgot to say this came out of another lint error about missing default case in the switch statement, so i just had this return the current state if the dispatch actions doesn't match any known action (such as adding a party member etc).